src/pages/namespace/Instances/List/index.tsx

```diff
@@ -37,7 +40,8 @@ const InstancesListPage = () => {
         {t("pages.instances.list.title")}
       </h3>
       <Card>
-        <Table>
+        <Filters filters={filters} onUpdate={setFilters} />
+        <Table className="border-t">
```

Comment:
I just noticed that the border color in dark mode is not correct (I guess it should match the table's outline)

This should fix it (I would also set the color explicitly for light mode as well): `<Table className="border-t border-gray-5 dark:border-gray-dark-5">`

 ![CleanShot 2023-07-25 at 13 35 31](https://github.com/direktiv/direktiv-ui/assets/121789579/225c7cee-2d5d-4c85-9ff9-749d5759ff0b)